docs/tracingpolicy/README.md

```diff
@@ -0,0 +1,1193 @@
+`char_buf`. So, `size_t` is the number of `char_buf` elements stored in the
+pointer of the second argument. Similarly, if we would like to capture the the
+`__x64_sys_writev(long, iovec *, vlen)` syscall, then `iovec` has a size of
+`vlen`, which is going to be the 3rd argument.
+
+Please note that `sizeArgIndex` is inconsistent at the moment and does not take
+the index, but the number of the index (or index + 1). So if the size is the
+third argument, index 2, the value should be 3.
+
+```yaml
+- call: "__x64_sys_write"
+  syscall: true
+  args:
+  - index: 0
+    type: "int"
+  - index: 1
+    type: "char_buf"
+    returnCopy: true
+    sizeArgIndex: 3
+  - index: 2
+    type: "size_t"
+```
+
+Note, that you can specify which arguments you would like to print from a
+specific syscall. For example if you don't care about the file descriptor,
+which is the first `int` argument with `index: 0` and just want the `char_buf`,
+what is written, then you can leave this section out and just define:
+```yaml
+args:
+- index: 1
+  type: "char_buf"
+  returnCopy: true
+  sizeArgIndex: 3
+- index: 2
+  type: "size_t"
+```
+This tells the printer to skip printing the `int` arg because it's not useful.
+
+### Selectors
+
+A `TracingPolicy` can contain from 0 to 5 selectors. A selector is composed of
+1 or more filters. The available filters are the following:
+- [`matchArgs`](#arguments-filter)
+- [`matchReturnArgs`](#return-args-filter)
+- [`matchPIDs`](#pids-filter)
+- [`matchBinaries`](#binaries-filter)
+- [`matchNamespaces`](#namespaces-filter)
+- [`matchCapabilities`](#capabilities-filter)
+- [`matchNamespaceChanges`](#namespace-changes-filter)
+- [`matchCapabilityChanges`](#capability-changes-filter)
+- [`matchActions`](#actions-filter)
+
+#### Arguments filter
+
+Arguments filters can be specified under the `matchArgs` field and provide
+filtering based on the value of the function's argument.
+
+In the next example, a selector is defined with a `matchArgs` filter that tells
+the BPF code to process only the function call for which the second argument,
+index equal to 1, concerns the file under the path `/etc/passwd` or
+`/etc/shadow`. It's using the operator `Equal` to match against the value of
+the argument.
+
+Note that conveniently, we can match against a path directly when the argument
+is of type `file`.
+
+```yaml
+selectors:
+- matchArgs:
+  - index: 1
+  operator: "Equal"
+  values:
+  - "/etc/passwd"
+  - "/etc/shadow"
+```
+
+**Further examples**
+
+In the previous example, we used the operator `Equal`, but we can also use the
+`Prefix` operator and match against all files under `/etc` with:
+```yaml
+selectors:
+- matchArgs:
+  - index: 1
+  operator: "Prefix"
+  values:
+  - "/etc"
+```
+In this situation, an event will be created every time a process tries to
+access a file under `/etc`.
+
+Although it makes less sense, you can also match over the first argument, to
```

Comment:
What is the use case for this?

Reply:
I think there is no use case, explaining the "Although it makes less sense" part, it's mostly to demonstrate the capabilities of the filter.